src/io/xpi.spec.ts

```diff
@@ -94,12 +94,12 @@ describe(__filename, () => {
   });
 
   const createXpi = ({
+    autoClose = true,
```

Comment:
Is there any case for `autoClose: false`? What would need to do that?

Reply:
mmh, for `autoClose: false` or `true`? `autoClose: false` means we can manually close the file after a bunch of read operations, which is a great improvement.